artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/Features/Timeline/GetProfileTimeLineService.cs

```diff
@@ -1,14 +1,69 @@
+using Artificial.Scrum.Master.ScrumIntegration.Exceptions;
+using Artificial.Scrum.Master.ScrumIntegration.Infrastructure.ApiTokens;
+using Artificial.Scrum.Master.ScrumIntegration.Infrastructure.ScrumServiceHttpClient;
+using Artificial.Scrum.Master.ScrumIntegration.Utilities;
+
 namespace Artificial.Scrum.Master.ScrumIntegration.Features.Timeline;
 
 internal interface IGetProfileTimeLineService
 {
-    Task<GetProfileTimeLineResponse> Handle(string userId);
+    Task<List<GetProfileTimeLineResponse>> Handle(string userId);
 }
 
 internal class GetProfileTimeLineService : IGetProfileTimeLineService
 {
-    public Task<GetProfileTimeLineResponse> Handle(string userId)
+    private readonly IUserTokensRepository _userTokensRepository;
+    private readonly IProjectHttpClientWrapper _projectHttpClientWrapper;
+    private readonly IJwtDecoder _jwtDecoder;
+
+    public GetProfileTimeLineService(IUserTokensRepository userTokensRepository,
```

Comment:
jakbyś mógł to proszę żebyś trochę inaczej formatował te argumenty? Bo to chyba resharpera robota i wygląda to tak se (jak w intellij xD). Według mnie (to moja osobista preferencja i możesz to mieć gdzieś) jak już chcesz ten parametry rozbijać na nowe linie to lepiej po jedynm na linie bo mi oczy uciekają w kosmos i nie mogę się skupić, trochę jak takie linq

```
list.Select(x => x).Where(x => x)
  .MaxBy(x => x.d) .ThenBy(x => x.dd).ToList()
```

Wiesz, kilka funkcji w jednej linii się trudniej czyta niż jak są w różnej. Tutaj jest di to mało ważne ale jakbyś miał duży konstruktor i nie ufał copilotowi, że dobrze wstrzyknął to łatwiej znaleźć :)